modules/openapi-generator/src/main/resources/dart2/api_client.mustache

```diff
@@ -66,10 +61,10 @@ class ApiClient {
     String path,
     String method,
     List<QueryParam> queryParams,
-    Object body,
+    Object? body,
     Map<String, String> headerParams,
```

Comment:
`headerParams` and `formParams` are nullable in other openapi generators, so I would prefer to keep it this way for language agnostic users